docs/concepts/distributed_trust.md

```diff
@@ -0,0 +1,85 @@
+---
+id: distributed_trust
+title: Distributed Trust
+---
+
+:::caution
+This feature is currently a preview on Peregrine and is therefore not available on Spiritnet.
+:::
+
+Attesters can be individuals that attest to the validity of claims made by claimers.
+But in many cases multiple attesters group together to build up a trust in a brand.
+Verifier don't need to trust individual attesters but trust the brand, which in return ensures that all attesters working for this brand are credible.
+
+Such a brand can be organized in many different ways.
+Today the most common way to organize and build a brand is to found a company.
+In the future their are going to be more decentralized solutions like Decentralized Autonomous Organizations (DAOs) or more specifically Virtual Credential Organizations (VCOs).
+
+If such a collective has given the attester the right to issue attestations, the attester can decide to create an attestation in the name of the collective.
```

Comment:
Seems a very long description, and I think it's best if it's not the very first thing a reader sees. We don't know yet how attestation works, but we are already talking about when someone leaves the collective.

Reply:
I think you are right. I wanted to explain a higher level concept so that the reader gets a to know the capabilities of the blockchain and later explain how the Delegation Hierarchies and VCOs implement the specifics.

But it's probably better to keep it even shorter. I would propose:

In the introduction we motivate why attesters should group up and organize themselves. But we don't explain how. After that we have more detailed sections for Delegation Hierarchies and VCOs. There we can go into more details.

[1932479](https://github.com/KILTprotocol/docs/pull/88/commits/19324793b0a4f5d0e9454f72c687d5d8ae82225e)